tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -170,6 +170,8 @@ def _test_shm_transfer_is_enabled(scheduler_commands, remote_command_executor, p
         result = scheduler_commands.submit_command("fi_info -p efa 2>&1 > /shared/fi_info.out")
     job_id = scheduler_commands.assert_job_submitted(result.stdout)
     scheduler_commands.wait_job_completed(job_id)
+    job_stdout = remote_command_executor.run_remote_command(f"cat /home/rocky/slurm-{job_id}.out").stdout
+    logging.info(f"Job stdout is: {job_stdout}")
```

Comment:
Minor: It might be better to make this a overall improvement in `wait_job_completed`